tests/fixtures/Middleware/proxy-ticket-test.yml

```diff
@@ -0,0 +1,229 @@
+-
+  ticket: DESKTOP_PT-21-9fp9
+  xml: |
```

Comment:
It's weird to mix XML inside yml.
Maybe we could have a path to a xml file:
xml: path/to/fixtures/xml/gimme.xml

Reply:
This is also possible... but then it will be more fixtures files to handle.
I'll leave you guys decide and I'll update accordingly.

Reply:
This will make fixtures hard to read, while if we keep all data in the same place it's easy to figure out what every test scenario does.

Reply:
Ok then, let's keep it as is.